bundles/org.openhab.voice.googlestt/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="voice:googlestt">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Authentication for connecting to Google Cloud Platform.</description>
+		</parameter-group>
+		<parameter-group name="stt">
+			<label>STT Configuration</label>
+			<description>Configure Speech to Text.</description>
+		</parameter-group>
+		<parameter-group name="messages">
+			<label>Info Messages</label>
+			<description>Configure service information messages.</description>
+		</parameter-group>
+		<parameter name="clientId" type="text" required="true" groupName="authentication">
+			<label>Client Id</label>
+			<description>Google Cloud Platform OAuth 2.0-Client Id.</description>
+		</parameter>
+		<parameter name="clientSecret" type="text" required="true" groupName="authentication">
+			<context>password</context>
+			<label>Client Secret</label>
+			<description>Google Cloud Platform OAuth 2.0-Client Secret.</description>
+		</parameter>
+		<parameter name="oauthCode" type="text" groupName="authentication">
+			<label>Authorization Code</label>
+			<description><![CDATA[The oauth code is a one-time code needed to retrieve the necessary access token from Google Cloud Platform. <b>Please go to your browser ...</b> https://accounts.google.com/o/oauth2/auth?client_id=\<YOUR_CLIENT_ID\>&redirect_uri=urn:ietf:wg:oauth:2.0:oob&scope=https://www.googleapis.com/auth/cloud-platform&response_type=code <b>... to generate an auth-code and paste it here</b>.]]></description>
+		</parameter>
+		<parameter name="singleUtteranceMode" type="boolean" required="true" groupName="stt">
```

Comment:
I assume `required="true"` is wrong as you provide a default value

Reply:
Sorry I think I miss understood this. I though I should require those to avoid the user to clean the values from the interface and that the default values were only used the first time the service is registered. Same apply to others.